### PR TITLE
Added task to copy and publish results

### PR DESCRIPTION
**Pull request by izavala**

### CODE REVIEW

**Comment by Aaronontheweb** [comment]
@izavala looks like I need to fix Incrementalist auth

File: build-system/azure-pipeline.template.yaml [hunk 1]
```diff
@@ -37,6 +37,26 @@ jobs:
           testResultsFiles: '**/*.trx' #TestResults folder usually
           testRunTitle: ${{ parameters.name }}
           mergeTestResults: true
+      - task: CopyFiles@2
+        displayName: Linux- Copy Files
+        inputs:
+          Contents: |
+            /home/vsts/work/1/s/TestResults/incrementalist-affected-folders.txt
```
Comment (Aaronontheweb) [inline build-system/azure-pipeline.template.yaml]: Ah, you need to use globbing here - not absolute paths. If they change anything with Azure Pipelines that will break this script.

File: build-system/azure-pipeline.template.yaml [hunk 2]
```diff
@@ -37,6 +37,26 @@ jobs:
...
+            /home/vsts/work/1/s/TestResults/incrementalist-affected-files.txt
+          TargetFolder: '$(Build.ArtifactStagingDirectory)'
+        condition: in( variables['Agent.OS'], 'Linux', 'Darwin' )
+      - task: CopyFiles@2
+        displayName: Windows- Copy Files 
+        inputs:
+          Contents: |
+            D:\a\1\s\TestResults\incrementalist-affected-folders.txt
```
Comment (Aaronontheweb) [inline build-system/azure-pipeline.template.yaml]: Same feedback as above.

Comment (izavala) [reply build-system/azure-pipeline.template.yaml]: @Aaronontheweb I changed this to not be an absolute path in the commit below? Would that be the correct way to do this?

**Comment by Aaronontheweb** [comment]
Nice work @izavala - this looks perfect.

**Pull request merged**